.storybook/preview.js

```diff
@@ -5,6 +5,7 @@ import { MockedProvider } from '@apollo/client/testing'
 // happo support
 import 'happo-plugin-storybook/register'
 
+import 'uswds/dist/css/uswds.css'
```

Comment:
By adding this import, all of the missing icons now appear 🌟 That includes all of the little black boxes in the News widget RSS feed